packages/vscode-dependencies-validation/src/diagnostics.ts

```diff
@@ -2,10 +2,13 @@ import type { DiagnosticCollection, Uri } from "vscode";
 import { Range, Diagnostic } from "vscode";
 import { isEmpty } from "lodash";
 import { findDependencyIssues } from "@sap-devx/npm-dependencies-validation";
+import { IChildLogger } from "@vscode-logging/types";
 import { getLogger } from "./logger/logger";
 import { NPM_DEPENDENCY_ISSUES_CODE } from "./constants";
 
-const logger = getLogger().getChildLogger({ label: "diagnostics" });
+export function logger(): IChildLogger {
```

Comment:
why is this exported?

Reply:
should it be exported via `internal` property if it is for tests purposes?

Reply:
Done